tools/integration-tests/tester/framework/framework.go

```diff
@@ -294,21 +287,13 @@ func (f *Framework) CreateDRNGNetwork(name string, members, peers int) (*DRNGNet
 		DRNGThreshold: 3,
 		DRNGDistKey:   hex.EncodeToString(drng.distKey),
 		DRNGCommittee: drngCommittee,
-		Mana:          true,
 		StartSynced:   true,
 	}
 
 	// create peers/GoShimmer nodes
 	for i := 0; i < peers; i++ {
 		config.Seed = privKeys[i].Seed().String()
-		if _, e := drng.CreatePeer(func() GoShimmerConfig {
-			if i == 0 {
```

Comment:
the faucetConfig is removed. Why is that - is it no longer needed?

Reply:
Because we don't need a faucet sending funds (usually we take `peers[0]` to be the faucet), thus `drng.CreatePeer` will not check `config.Faucet` to specifically set up the faucet wallet.
There's no need to set the `config.Faucet` here, so just take the config above works.